src/presentation/controllers/device-controller.ts

```diff
@@ -33,6 +35,24 @@ export class DeviceController extends BaseController {
     }
   }
 
+  private toDeviceDto(device: Device): DeviceDtoInterface | undefined {
+    if (this.isINewsDevice(device)) {
+      return new INewsDeviceDto(device)
+    }
+    if (this.isTelemetricsDevice(device)) {
+      return new TelemetricsDeviceDto(device)
+    }
+    return undefined
+  }
```

Comment:
What do you think of this style? 😄 

```suggestion
  private toDeviceDto(device: Device): DeviceDtoInterface | undefined {
    if (this.isINewsDevice(device)) return new INewsDeviceDto(device)
    if (this.isTelemetricsDevice(device)) return new TelemetricsDeviceDto(device)
    return undefined
  }
```